loki-logback-appender/src/main/java/com/github/loki4j/logback/AbstractLoki4jEncoder.java

```diff
@@ -243,6 +250,9 @@ String[] extractStreamKVPairs(String stream) {
                 result[pos + 1] = kv[1];
                 pos += 2;
             } else {
+                if (label.ignoreBlankLabel) {
```

Comment:
The condition `kv.length != 2` does not necessary mean empty label value, it could be a parsing error as well.

Reply:
You are right, maybe I can rename this option to "ignoreParsingErrors". I don't think the labels are the most important information for logs, and in this part i try to prevent dropping logs because of any label parsing error.

Reply:
I mean provide two options 'ignoreBlankLabel' and 'ignoreParsingError'

Reply:
I mean maybe we can provide two options 'ignoreBlankLable' and 'ignoreLabelParsingError'